scipy/stats/tests/test_generation/reference_distributions.py

```diff
@@ -92,15 +92,21 @@ def _cdf(self, x, **kwargs):
         if ((self._cdf.__func__ is ReferenceDistribution._cdf)
                 and (self._sf.__func__ is not ReferenceDistribution._sf)):
             return mp.one - self._sf(x, **kwargs)
-        a, _ = self._support(**kwargs)
-        return mp.quad(lambda x: self._pdf(x, **kwargs), (a, x))
+
+        a, b = self._support(**kwargs)
+        res = mp.quad(lambda x: self._pdf(x, **kwargs), (a, x))
+        res = res if res < 0.5 else mp.one - self._sf(x, **kwargs)
```

Comment:
This has come up a few times before that the reference distribution SF/CDF were not as accurate as they could be because they integrate over too much range. I thought I'd take the opportunity to fix it here.